src/commands/publish/index.js

```diff
@@ -0,0 +1,122 @@
+const fs = require("fs-extra");
+const chalk = require("chalk");
+const path = require("path");
+const Zip = require("adm-zip");
+const prompts = require("prompts");
+const paths = require("../../utils/paths");
+const manifestValidator = require("./manifest-validator");
+const apiClient = require("./zeplin-api");
+
+const pathResolver = {
+    init(root) {
+        this.root = root;
+    },
+    resolve(relativePath) {
+        if (this.root) {
+            return path.resolve(this.root, relativePath);
+        }
+
+        return paths.resolveBuildPath(relativePath);
+    }
+};
+
+function parseManifest() {
+    const manifestPath = pathResolver.resolve("./manifest.json");
+
+    if (!fs.existsSync(manifestPath)) {
+        throw new Error("Could not find manifest.json file!");
```

Comment:
> Locating **manifest.json** failed.

Both for this and the error below, could we suggest something like re-building? What could typically go wrong for these errors?